test/libponyc-run/runner/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ add_custom_command(OUTPUT ${RUNNER_EXECUTABLE}
     main.pony
     _build_process_notify.pony
     _cli_options.pony
+    _colors.pony
```

Comment:
what's this change?

Reply:
While looking at how the runner works i saw that this file was missing from the list of dependent files.

Reply:
Can you do this in another PR as it is unrelated?